service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java

```diff
@@ -295,36 +294,20 @@ private String getCourtName(String localAuthorityCode) {
         return formattedDirections.build();
     }
 
-    private List<Map<String, Object>> getFormattedOtherPartiesDirections(
-        Map<OtherPartiesDirectionAssignee, List<Element<Direction>>> groupedOtherParties) {
+    private List<Map<String, Object>> getFormattedDirections(
+        Map<LabelledAssignee, List<Element<Direction>>> groupedDirections) {
 
-        List<Map<String, Object>> directionsToOthers = new ArrayList<>();
-        groupedOtherParties.forEach((key, value) -> {
-            Map<String, Object> directionForOthers = new HashMap<>();
-            directionForOthers.put("header", "For " + key.getLabel());
+        List<Map<String, Object>> directions = new ArrayList<>();
+        groupedDirections.forEach((key, value) -> {
```

Comment:
if you are here it would be nice to rename (key, value)  to (assignee, assigneeDirection)

Reply:
done 👍